Dockerfile

```diff
@@ -9,7 +9,13 @@ ARG UID=101
 LABEL maintainer="Alejandro Celaya <alejandro@alejandrocelaya.com>"
 USER root
 RUN rm -r /usr/share/nginx/html && rm /etc/nginx/conf.d/default.conf
-USER $UID
 COPY config/docker/nginx.conf /etc/nginx/conf.d/default.conf
 COPY scripts/docker/servers_from_env.sh /docker-entrypoint.d/30-shlink-servers-json.sh
 COPY --from=node /shlink-web-client/build /usr/share/nginx/html
+
+# Allow the nginx user to create new files in the html dir, but not modify existing files.
+# This is required by 30-shlink-servers-json.sh
```

Comment:
I think this comment is misleading, as the user can still modify files directly in the `html` folder, but not its subfolders.

Reply:
Correct, seems I was a bit too tired at the time and wrongly assumed dir level write would not confer edit, but indeed it does. Switched to creating an empty server list by default.